parser/proto.y

```diff
@@ -753,9 +753,8 @@ msgReserved : _RESERVED tagRanges ';' {
 	}
 	| reservedNames
 
-enumReserved : _RESERVED enumValueRanges semicolons {
-		semi, extra := protolex.(*protoLex).requireSemicolon($3)
-		$$ = newNodeWithEmptyDecls(ast.NewReservedRangesNode($1.ToKeyword(), $2.ranges, $2.commas, semi), extra)
+enumReserved : _RESERVED enumValueRanges ';' semicolons {
```

Comment:
nit: let's add a TODO comment explaining why the `';'` is here.

Reply:
Done